client_sdk/avalon_client_sdk/quorum/contracts/WorkOrder.sol

```diff
@@ -0,0 +1,172 @@
+pragma solidity >=0.4.0;
+
```

Comment:
We already have workorder contract in ethereum folder, can't we use that contract for quorum?

Reply:
There are some modifications that are required by the Coldchain application.